Api 17275 update poa pdf

acovrig opened the pull request:
<!-- Please read our guidelines before submitting your first PR https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/engineering/code_review_guidelines.md -->

## Description of change
<!-- Please include a description of the change and context. What would a code reviewer, or a future dev, need to know about this PR in order to understand why this PR is necessary? This could include dependencies introduced, changes in behavior, pointers to more detailed documentation. The description should be more than a link to an issue.  -->
Add `SPACE FORCE` to the 2122a endpoint and update the PDF to match.

## Original issue(s)
[API-17275](https://vajira.max.gov/browse/API-17275)

## Things to know about this PR
<!--
* Are there additions to a `settings.yml` file? Do they vary by environment?
* Is there a feature flag? What is it?
* Is there some Sentry logging that was added? What alerts are relevant?
* Are there any Prometheus metrics being collected? What Grafana dashboard were they added do?
* Are there Swagger docs that were updated?
* Is there any PII concerns or questions?
-->
* I tested the PDF generation/filling, I'm not sure what the right way is to make the `1.pdf` - I took the official PDF and used Acrobat to export just page 1.

<!-- Please describe testing done to verify the changes or any testing planned. -->

acovrig commented:
This should not be merged until release notes are created!

jw81 commented on modules/claims_api/spec/lib/claims_api/poa_pdf_constructor/individual_spec.rb:
```diff
@@ -45,7 +45,7 @@
   end
 
-  xit 'construct pdf' do
+  it 'construct pdf' do
```
We shouldn't re-enable this spec as part of this PR.
Somebody disabled it in the past because it was flaky.
If we want to re-enable it, we should create a ticket to go investigate if it's ok to re-enable and make that change (and only that change).

jw81 reviewed (changes requested):
Asked for a change and I want to personally dedicate some time to playing around with this and see that it's working as expected.

jw81 commented:
Do we know why/how these files got added?
If we don't need them for some reason, we shouldn't added them to the repo.
`modules/claims_api/spec/fixtures/VBA-21-22-ARE.pdf`
`modules/claims_api/spec/fixtures/VBA-21-22A-ARE.pdf`

jw81 commented:
> I added 2 things to the seeds, I'm not sure if that's OK.
Not sure I follow what you mean here.  🤔 

> I also noticed the phone number had a dash in it (in the example) and that doesn't match the regex.
It looks like you reverted that change because the team agreed we should handle that outside of this PR/ticket.  So can you remove this bullet from the PR, or cross it out for historical purposes?

acovrig commented:
> Do we know why/how these files got added? If we don't need them for some reason, we shouldn't added them to the repo. `modules/claims_api/spec/fixtures/VBA-21-22-ARE.pdf` `modules/claims_api/spec/fixtures/VBA-21-22A-ARE.pdf`

I updated one of them and added the other - the official PDF got pages added and a checkbox added, and I added the other PDF for completeness.

jw81 commented:
> > Do we know why/how these files got added? If we don't need them for some reason, we shouldn't added them to the repo. `modules/claims_api/spec/fixtures/VBA-21-22-ARE.pdf` `modules/claims_api/spec/fixtures/VBA-21-22A-ARE.pdf`
> 
> I updated one of them and added the other - the official PDF got pages added and a checkbox added, and I added the other PDF for completeness.

Ah ok.  
I didn't notice that the `VBA-21-22A-ARE.pdf` already existed.  Ok, yeah prob a good idea then to add the other for completeness like ya did.  👍

jw81 reviewed (approved):
I was able to play with the code locally and confirm that the generated form is selecting the new "Space Force" checkbox and everything else still appears to be functioning properly.

Nice job!!!  🥳
